src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -122,6 +122,9 @@ const iconStyles: ComponentSlotStylesInput<IconProps, any> = {
   path: getSvgStyle('path'),
 
   secondaryPath: getSvgStyle('secondaryPath'),
+
+  unfilled: getSvgStyle('unfilled'),
```

Comment:
I am implementing this in my PR where we won't need to have these styles defined in each SVG. I also named them filled & outline. Do you think filled & unfilled works better?

Reply:
I think `filled` & `outline` are fine, `unfilled` was used in legacy client.